CI/physmon/physmon.py

```diff
@@ -175,21 +177,27 @@
             trackingGeometry,
             field,
             CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
+            TrackSelectorRanges(
+                removeNeutral=True,
+                absEta=(None, 2.5),
+                loc0=(None, 4.0 * u.mm),
+                pt=(500 * u.MeV, None),
+            ),
```

Comment:
What's the default here? Why don't we go higher and select like before?

Reply:
I just moved this from the vertexing to the CFK so it should do the same as before